wd_core.au3

```diff
@@ -1259,55 +1259,54 @@ Func _WD_Startup()
 	$sFile = __WD_StripPath($_WD_DRIVER)
 	$iPID = ProcessExists($sFile)
 
+	Local $bErrorOccured = False

```

Comment:
I plan in next step change `$bErrorOccured` to commonly used `$iErr`

Reply:
I haven't tested it, but this appears to be more inline with my expectations.

Reply:
Before I do next step I will wait on your pre-aproval

Reply:
If your "next step" is simple to rename the variable, then I would say it's fine to proceed. If it's something more complex, then you may want to provide further details here before proceeding.

Reply:
Done